NEWS.md

```diff
@@ -117,6 +117,15 @@ be directly added to this file to describe the related changes.
   module, and `test_plant_model` (in `crop_model_testing_helper_functions.R`)
   now uses `expect_no_error` instead of `expect_silent`.
 
+- Changed the energy_par_content constant to 0.219. This is a conversion rate 
+  from photons to W/m2, which equals 1/4.57. 4.57 is a commonly used constant 
+  to convert PAR in W/m2 to micromol m-2 s-1. Users should be cautious when 
+  performing this conversion outside of BioCro to make sure the two are the same,
+  for example, pre-processing the radiation data. See more details in 
```

Comment:
I'm going to try rewording the middle two sentences, partly to make sure I'm really understanding this correctly, and (if I am) partly to make it clearer to readers.

"This is a conversion rate from photons (in micromoles) to watt-seconds.  It equals 1/4.57, 4.57 being a commonly used constant to convert PAR in W/m2 to micromol m-2 s-1.  Users should take care to ensure that if preprocessing of radiation data is required to prepare it for use with BioCro, the same conversion factor is used."

Probably this information should go somewhere else in addition to (or instead of?) being hidden away in the news file.  But that can be a separate issue.

Reply:
If we are going to modify this, I would also like to explain the parameter's meaning more clearly. To me a conversion factor is something that only changes the units, e.g. gallons of water to liters of water. The `par_energy_content` factor is different, because it relates the energy of light to the number of photons. In the water analogy, it would be more like the density of water, which could be used if we wanted to specify a weight of water rather than a volume. We're still specifying the amount of water, but using a totally different type of measurement.

"This factor represents the average energy per photon in photosynthetically active radiation (PAR), expressed in units of Joules (or, equivalently, Watt-seconds) per micromole of photons. This value of this ratio depends on the spectrum of radiation in the PAR range. The value used here equals 1/4.57, 4.57 being a commonly used constant to convert PAR in W/m2 to micromol m-2 s-1. Users should take care that this factor is used consistently if radiation data is preprocessed for use with BioCro."

Reply:
Here's a further change to that using more exact terminology. I find that a phrase like "converting photons to W m^-2" doesn't make sense.

"This is a conversion rate from photon flux density (in micromoles per square meter per second) to energy flux (joules per square meter per second, also watts per square meter) for photosynthetically active radiation (PAR). It equals 1/4.57, 4.57 being a commonly used constant to convert PAR in W m^-2 to micromole m^-2 s^-1. Users should take care to ensure that if processing of radiation data is required to prepare it for use with BioCro, the same conversion factor is used."

Reply:
I like Ed's, but "joules" and "watt" aren't capitalized and "^" for exponentiation.

Reply:
Thanks for all the suggestions. I find Justin's version clearer. It's Watt (=Joule/second), but not watt-seconds.

Reply:
> It's Watt (=Joule/second), but not watt-seconds.

Well, for what it's worth, it seems to me you could either say

1 μmol (photons) = 0.219 J = 0.219 watt-seconds

or

1 μmol/m^2/s = 0.219 J/m^2/s = 0.219 W/m^2

In any case, the new wording you committed seems entirely suitable (unless review fatigue has caused me to overlook something), and I will submit a new review giving explicit approval in case that's what I need to do to remove the block on merging.